packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -132,6 +132,18 @@ function ViewGroupInner(
     setGroupImage,
   } = useGroup(accountId, chat)
 
+  useEffect(() => {
+    onDCEvent(accountId, 'ContactsChanged', () => {
```

Comment:
```suggestion
    return onDCEvent(accountId, 'ContactsChanged', () => {
```

The event listerner needs to be cleaned up.